src/metadata/routes.clj

```diff
@@ -18,27 +17,27 @@
 
 (defapi app
   {:exceptions cx/exception-handlers}
-  (swagger-ui config/docs-uri
-    :validator-url nil)
-  (swagger-docs
-    {:info {:title "Discovery Environment Metadata API"
-            :description "Documentation for the Discovery Environment Metadata REST API"
-            :version "2.0.0"}
-     :tags [{:name "service-info", :description "Service Information"}
-            {:name "avus", :description "Attribute/Value/Unit Management"}
-            {:name "permanent-id-requests", :description "Permanent ID Request Management"}
-            {:name "admin-permanent-id-requests", :description "Admin Permanent ID Request Mgmt"}
-            {:name "data-comments", :description "Comments on Data Items"}
-            {:name "app-comments", :description "Comments on Apps"}
-            {:name "favorites", :description "Favorite Resources"}
-            {:name "ontologies", :description "Ontology Hierarchies"}
-            {:name "admin-ontologies", :description "Ontology Hierarchy Administration"}
-            {:name "admin-data-comments", :description "Admin Data Item Comment Management"}
-            {:name "admin-app-comments", :description "Admin App Comment Management"}
-            {:name "tags", :description "Tags Management"}
-            {:name "template-info", :description "Template Information"}
-            {:name "template-administration", :description "Template Administration"}]})
-  (middlewares
+  (swagger-routes
+    {:ui      config/docs-uri
+     :options {:ui {:validatorUrl nil}}
+     :data    {:info {:title "Discovery Environment Metadata API"
+                      :description "Documentation for the Discovery Environment Metadata REST API"
+                      :version "2.8.0"}
```

Comment:
That was my thought too. This version hasn't been updated in a while, even though the endpoints have been updated since DE 2.0, so I thought it should be updated to at least 2.8.0 (the last time endpoints were updated).

If we want to keep these APIs versioned, it makes sense to me to just update it to the version in project.clj whenever an endpoint is updated, but if we don't want to keep these versioned, we can also probably just remove the `:version` from these (they're not public after all).